lib/ash_ai/tools.ex

```diff
@@ -433,6 +433,7 @@ defmodule AshAi.Tools do
           input: %{
             type: :object,
             properties: properties,
+            additionalProperties: false,
```

Comment:
I think there are cases where this will cause problems with some providers? I'm not sure. What providers have you tested this with?